depthmapX/GraphDoc.cpp

```diff
@@ -938,7 +938,7 @@ void QGraphDoc::OnEditGrid()
    }
    bool newmap = false;
 
-   if (!m_meta_graph->PointMaps::size() || m_meta_graph->getDisplayedPointMap().isProcessed()) {
+   if (!m_meta_graph->PointMaps::maps_vector.size() || m_meta_graph->getDisplayedPointMap().isProcessed()) {

```

Comment:
That fact that you have to write PointMaps::maps_vector in a file talking to the MetaGraph class is a fairly strong code smell - I know it's been there before, but that just means the old code wasn't good, and we want to improve it when we touch it...